loadgen/README.md

```diff
@@ -103,3 +99,67 @@ with the reference models.
+</div>
+
+The loadgen implementation is common to all submissions, while the QSL (“Query Sample Library”) and SUT (“System Under Test”) are implemented by submitters. QSL is responsible for loading the data and includes untimed preprocessing.
+
+A submission over the network introduces a new component “QDL” (query dispatch library) and a it would look like this:
+
+<div align="center" style="display:flex; flex-flow:row wrap; justify-content: space-evenly;">
+<img src="diagram_network_submission.png" width="300px" style="padding: 20px">
+</div>
+
+QDL is a proxy for a load-balancer, that dispatches queries to SUT over a physical network, receives the responses and passes them back to loadgen.  It is implemented by the submitter. The interface of the QDL is the same as the API to SUT. 
+
+In scenarios using QDL, data in QSL may be compressed at the choice of the submitter in order to reduce network transmission time. Decompression is part of the timed processing in SUT. A set of approved standard compression schemes will be specified for each benchmark; additional compression schemes must be approved in advance by the Working Group.
+
+All communication between loadgen/QSL and SUT is via QDL, and all communication between QDL and SUT must pass over a physical network.
+
+Query batching can be implemented either in QDL or SUT, or split between the two components, at the submitter’s option.  Current Plan of record is no batching in QDL. QDL batching would be considered for future releases. Aside from batching, QDL implements only the protocol to transmit queries over the network and receive responses. Performing any part of the timed preprocessing or inference in QDL is specifically disallowed.
+
+The MLperf over the Network will run in Server mode and Offline mode. All loadgen modes are expected to work as is with insignificant changes. These include running the test in performance mode, accuracy mode, find peak performance Search mode and compliance mode. The same applies for power measurements.
+
+### QDL details
+The Query Dispatch Library is a library implemented by the submitter and provides the Loadgen the same SUT API. All MLPerf Inference SUTs implement the `mlperf::SystemUnderTest` class which is defined in system_under_test.h. The QDL should follow the same API, support all existing `mlperf::SystemUnderTest` methods and use the same h file.
+
+#### QDL Query issue and response over the network
+
+The QDL gets the queries from the loadgen through 
+```CPP
+void IssueQuery(const std::vector<QuerySample>& samples)
+```
+
+The QDL should dispatch the queries to the SUT over the physical media. The exact method and implementation for it are submitter specific and would not be specified at MLCommons. Submitter implementation should include all methods required to serialize the query, load balance, drive it to the Operating system and network interface card and send to the SUT.
```

Comment:
Having read this far... do we need to do a deduplication pass to ensure that the same info is not present in the loadgen and the inference_policies documents? Having information in two places is a recipe for it getting out of step.

Reply:
+1

Reply:
yea I don't think it is a great duplication and it is making a story clearer.

Reply:
Removed normative language to make clear this is only a description of the architecture.